tools/nuget/nuget.proj

```diff
@@ -35,7 +35,7 @@
       <None Include="..\..\include\ebpf_netevent_program_attach_type_guids.h" PackagePath="build\native\include" Pack="true" />
       <None Include="..\..\$(Platform)\$(Configuration)\netevent_ebpf_ext_export_program_info.exe" PackagePath="build\native\bin" Pack="true" />
       <None Include="..\..\$(Platform)\$(Configuration)\netevent_ebpf_ext_export_program_info.pdb" PackagePath="build\native\bin" Pack="true" />
-      <None Include="..\..\scripts\Install-Extension.ps1" PackagePath="build\native\bin" Pack="true" />
+      <None Include="..\..\$(Platform\$(Configuration)scripts\Install-Extension.ps1" PackagePath="build\native\bin" Pack="true" />
```

Comment:
Looks like a typo here, missing end bracket on `Platform` MSBuild property:

```suggestion
      <None Include="..\..\$(Platform)\$(Configuration)scripts\Install-Extension.ps1" PackagePath="build\native\bin" Pack="true" />
```

Reply:
Interesting - will queue an internal run prior to completing this change.

Reply:
Sorry about that - there were some errors in the post-build.ps1, but the pipeline showed green. I have resolved the issues now and confirmed that the created package contains the necessary script and no complaints about the signing.